kml/serdes.cpp

```diff
@@ -120,8 +120,9 @@ std::string GetStyleForPredefinedColor(PredefinedColor color)
   case PredefinedColor::Orange: return "placemark-orange";
   case PredefinedColor::None:
   case PredefinedColor::Count:
-    return {};
+    break;
   }
+  return {};
```

Comment:
I think we need an assertion here too. @rokuz ?

Reply:
Well, not for the ::None case I expect.
If for the ::Count and any special cases then maybe it would be better to first change/fix the code, to not have bug fixes mixed in warning fixes?
I admit I did these changes in the assumption that you are relying on the "... not handled in switch" warnings to make sure all cases are updated if the enum is expanded, as runtime errors are really bad for handling this anyway.

Reply:
@mpimenov As I remember we intentionally do not put default 'return' in such methods. It helps to tracks new values in enums.
@rdoeffinger I recommend to put CHECK(false, ("Incorrect enum value")); instead of return {}

Reply:
@rokuz I thought so too but couldn't find an example. I'm ok with CHECK instead of return.